operator/pkg/types/installs.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 // RenderSrc offers utility methods for processing of manifest resources.
+//
+//nolint:interfacebloat
```

Comment:
This lint is valid. The RenderSrc is really getting complex now. I think we should think wether its possible to somehow reduce the interface.

Reply:
The lint toleration is gone after I removed trasnformer since it's a generic implementation. But we should decide which functions we do not need here as a followup.